apps/i18n/common/en_us.json

```diff
@@ -493,6 +494,9 @@
   "tooMuchWork": "You made me do a lot of work!  Could you try repeating fewer times?",
   "toolboxHeader": "Blocks",
   "toolboxHeaderDroplet": "Toolbox",
+  "topCourseExplanation": "Continue lesson to jump to where you left off or view the whole course to see an overview of your progress.",
+  "topCourseLessonIntro": "You are currently working on",
```

Comment:
When combining strings, always have the complete sentence in this file, and have a substitution point (like `{puzzleNumber}` above inside the sentence.  That's because not all languages will want the other part(s) of the sentence to be added to the end.